src/ducks/settings/AccountsSettings.styl

```diff
@@ -18,10 +18,18 @@
     &__bank
         maxed-flex-basis 20%
 
+        +small-screen
+            maxed-flex-basis 33%
+
     &__number
         maxed-flex-basis 15%
         color var(--coolGrey)
 
+        +small-screen
```

Comment:
I thought we had to use `+small-screen()` (with parenthesis), are you sur this is working ?